### ISSUE TITLE
CW key referenced in TOF action

### ISSUE THREAD

**Issue by briantoby** [AdvancedPhotonSource/GSAS-II#58]
Hi Brian,
 
I’m continuing on to run a calibration and getting a crash on a key error:
 ```
Traceback (most recent call last):
  File "/SNS/users/66j/miniconda3-new/envs/GSASII/GSAS-II/GSASII/GSASIIpwdGUI.py", line 2438, in OnCalibrate
    if G2pwd.DoCalibInst(IndexPeaks,data,Sample):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/SNS/users/66j/miniconda3-new/envs/GSASII/GSAS-II/GSASII/GSASIIpwd.py", line 2325, in DoCalibInst
    GetInstParms(parmDict,varyList)
  File "/SNS/users/66j/miniconda3-new/envs/GSASII/GSAS-II/GSASII/GSASIIpwd.py", line 2241, in GetInstParms
    Sample[name][0] = parmDict[name]
                      ~~~~~~~~^^^^^^
KeyError: 'DisplaceX'
```

**Comment by briantoby** [comment]
addressed in #9a668ac5

**Comment by briantoby** [comment]
More work: TOF calibrate generates an infinite loop if the refinement is singular. Fixed in 9ce337e5
